onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -21,6 +21,11 @@ Module Name:
 // Quantized integer matrix/matrix dispatch structure.
 //
 
+struct MLAS_GEMM_U8X8_PARAMETERS {
```

Comment:
with the shape/data split apart now, is this struct still necessary? before, we had talked about the batched API taking two types of structures that would be glued together to make a parameters block, but if everything is already using parameters, then maybe the workers should just take two pointers instead to avoid the copy

Reply:
It definitely looks better. I will make the change and remove this.
Though essentially the worker's parameters would be encapsulated in a lambda object, which is essentially a struct with two pointers, and then when we use it, it becomes two level pointer chasing